modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DEImageField.testcase

```diff
@@ -115,7 +115,7 @@ definition {
 
 		WebContentNavigator.gotoEditCP(webContentTitle = "Web Content Title");
 
-		DERenderer.clearImageField(fieldLabel = "Image");
+		DERenderer.clearField(fieldLabel = "Image");
```

Comment:
@guivilarim, why did you decide to fix a test inside a pull request intended to create new tests only? Was this fix mandatory to the tests being created by this pull succeed? If not, honestly, I believe that we should create a specific pull to tackle the test fix.